app/webpacker/components/Results/Rankings/RankingsTable.jsx

```diff
@@ -0,0 +1,186 @@
+import React, { useMemo } from 'react';
+import { Table } from 'semantic-ui-react';
+import _ from 'lodash';
+import { flexRender, getCoreRowModel, useReactTable } from '@tanstack/react-table';
+import I18n from '../../../lib/i18n';
+import { formatAttemptResult } from '../../../lib/wca-live/attempts';
+import CountryFlag from '../../wca/CountryFlag';
+import { continents, countries } from '../../../lib/wca-data.js.erb';
+import { personUrl } from '../../../lib/requests/routes.js.erb';
+
+function CountryCell({ country }) {
+  return (
+    <Table.Cell textAlign="left">
+      {country.iso2 && <CountryFlag iso2={country.iso2} />}
+      {' '}
+      {country.name}
```

Comment:
:warning: unreliable, see my other remark above.

Reply:
nope, this is actually already a country that comes from the frontend